app/utils/incident/IncidentAnswersHelper.scala

```diff
@@ -172,7 +172,7 @@ class IncidentAnswersHelper(
 
   def transportMeansRegisteredCountry: Option[SummaryListRow] = getAnswerAndBuildRow[Nationality](
     page = TransportNationalityPage(incidentIndex),
-    formatAnswer = formatAsText,
+    formatAnswer = _.toString.toText,
```

Comment:
Why not just use the formatAsCountry formatter?

Reply:
This one is a different data type, it’s using the Nationality model